src/main/resources/liquibase/changelog/20190725115021_changelog.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet author="user (generated)" id="1564048228083-6">
+        <addColumn tableName="field_config">
+            <column name="field_type" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet author="user (generated)" id="1564048228083-7">
+        <addColumn tableName="field_config">
+            <column name="sort_order" type="integer">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet author="user (generated)" id="1564048228083-8">
+        <dropColumn columnName="order" tableName="field_config"/>
+    </changeSet>
+    <changeSet author="user (generated)" id="1564048228083-9">
+        <dropColumn columnName="type" tableName="field_config"/>
+    </changeSet>
+    <changeSet author="user (generated)" id="1564048228083-1">
```

Comment:
when you generate a changelog it will always create those changesets that recreate the unique keys, just remove those
this is a liquibase-hibernate5 known bug